src/FormFeedback/Tooltip/helpers/__tests__/tooltipArrowStyle.test.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import tooltipArrowStyle from "../tooltipArrowStyle";
+import resolveColor from "../resolveColor";
+
+const resultWithColor = (width, color) =>
+  `border-width:${width};border-color:${color};`.replace("__color__", resolveColor.toString());
+
+describe("tooltipArrowStyle", () => {
+  it("top position", () => {
+    expect(tooltipArrowStyle({ position: "top" }).join("")).toEqual(
+      resultWithColor("7px 7px 0 7px", "__color__ transparent transparent transparent"),
+    );
+  });
+
+  it("bottom position", () => {
+    expect(tooltipArrowStyle({ position: "bottom" }).join("")).toEqual(
+      resultWithColor("0 7px 7px 7px", "transparent transparent __color__ transparent"),
+    );
+  });
+});
```

Comment:
I recommend this approach for all tests of this type (`resolveTooltipPosition` etc.):

```js
// @flow
import tooltipArrowStyle from "../tooltipArrowStyle";

jest.mock("../resolveColor", () => "<computed>");

describe("tooltipArrowStyle", () => {
  it("top position", () => {
    expect(tooltipArrowStyle({ position: "top" }).join("")).toMatchInlineSnapshot(
      `"border-width:7px 7px 0 7px;border-color:<computed> transparent transparent transparent;"`,
    );
  });

  it("bottom position", () => {
    expect(tooltipArrowStyle({ position: "bottom" }).join("")).toMatchInlineSnapshot(
      `"border-width:0 7px 7px 7px;border-color:transparent transparent <computed> transparent;"`,
    );
  });
});
```

Generally avoid computing the expected value.